SpatialGDK/Source/SpatialGDKFunctionalTests/Private/SpatialFunctionalTest.cpp

```diff
@@ -36,6 +36,27 @@ constexpr float FINISH_TEST_GRACE_PERIOD_DURATION = 2.0f;
 ASpatialFunctionalTest::ASpatialFunctionalTest()
 	: Super()
 	, FlowControllerSpawner(this, ASpatialFunctionalTestFlowController::StaticClass())
+	, bIsStandaloneTest(false)
+{
+	Init();
+}
+
+ASpatialFunctionalTest::ASpatialFunctionalTest(const EMapCategory MapCiCategory, const int32 NumberOfClients /*=1*/,
+											   const FVector& InTestPositionInWorld /*=FVector::ZeroVector*/)
+	: Super()
+	, FlowControllerSpawner(this, ASpatialFunctionalTestFlowController::StaticClass())
+	, bIsStandaloneTest(true)
+	, TestPositionInWorld(InTestPositionInWorld)
```

Comment:
Is it important to be able to set this to non zero? I can't see a good use case for this to be anything but zero?